src/uu/stty/src/flags.rs

```diff
@@ -0,0 +1,199 @@
+// * This file is part of the uutils coreutils package.
+// *
+// * For the full copyright and license information, please view the LICENSE file
+// * that was distributed with this source code.
+
+// spell-checker:ignore parenb parodd cmspar hupcl cstopb cread clocal crtscts CSIZE
+// spell-checker:ignore ignbrk brkint ignpar parmrk inpck istrip inlcr igncr icrnl ixoff ixon iuclc ixany imaxbel iutf
+// spell-checker:ignore opost olcuc ocrnl onlcr onocr onlret ofill ofdel nldly crdly tabdly bsdly vtdly ffdly
+// spell-checker:ignore isig icanon iexten echoe crterase echok echonl noflsh xcase tostop echoprt prterase echoctl ctlecho echoke crtkill flusho extproc
+
+use crate::Flag;
+use nix::sys::termios::{
+    BaudRate, ControlFlags as C, InputFlags as I, LocalFlags as L, OutputFlags as O,
+};
+
+pub const CONTROL_FLAGS: [Flag<C>; 12] = [
+    Flag::new("parenb", C::PARENB),
+    Flag::new("parodd", C::PARODD),
+    Flag::new("cmspar", C::CMSPAR),
```

Comment:
I guess you saw this error?

Reply:
@tertsdiepraam ^^(in case you missed it :)

Reply:
I did see it, but I'm on holiday at the moment so I'm unable to fix it 😄 I'll take a look wt it in a few weeks